resolver/oneInchmulti.js

```diff
@@ -0,0 +1,27 @@
+import axios from "axios";
+export async function ExecuteQuote(quoteParams) {
```

Comment:
Params should be defined here only. Don't use pre-defined structs.

Reply:
Should I pass params here as Array after splitting it or pass em as string and split them here?